web/main.js

```diff
@@ -553,10 +570,14 @@ function hashchange() {
 }
 function codechange() {
     if (phraseInput.value === "") {
-        dialButton.value = "CREATE WORMHOLE";
+        dialButton.value = "Get file";
+        dialButton.disabled = true;
+        dialButton.hidden = false;
     }
     else {
-        dialButton.value = "JOIN WORMHOLE";
+        dialButton.value = "Get file";
+        dialButton.disabled = false;
+        dialButton.hidden = false;
```

Comment:
We repeated these two lines (setting the values to `true` or `false`) in many places. Maybe it's worth creating a function to show/hide `dialButton`.

Reply:
A function setDialButtonEnabled has been created.